### ISSUE TITLE
Use correct ARIA attributes in Vega

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1076]
In the multimapdialog (and potentially elsewhere), the attribute aria-labelledby is used when it should be aria-label (the first points to a specific element to use as a label, the second supplies the string label instead)

https://github.com/SatcherInstitute/health-equity-tracker/blob/abdb87e56f5fa3e18bf01ccc6afc95a39d11b5b1/frontend/src/cards/ui/MultiMapDialog.tsx#L60
